src/test/java/integration/ConflictAIT.java

```diff
@@ -0,0 +1,44 @@
+package integration;
+
+import java.io.IOException;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.coveros.selenified.output.Assert;
+import com.coveros.selenified.selenium.Action;
+import com.coveros.selenified.selenium.Element;
+import com.coveros.selenified.selenium.Selenium.Locator;
+import com.coveros.selenified.tools.TestBase;
+
+
+public class ConflictAIT extends TestBase {
+
+	@BeforeClass(alwaysRun = true)
+	public void beforeClass() throws IOException {
```

Comment:
We bo longer need to throw the ioexception. In fact, because you are, it's not overridding the beforeClass method